sem/manager.py

```diff
@@ -318,6 +348,31 @@ def run_simulations(self, param_list, show_progress=True, stop_on_errors=True):
         if param_list == []:
             return
 
+            # check
+
+        # If logging is enabled, convert the passed log_component
+        # dictionary or string to NS_LOG environment variable format
+        environment = {}
+        if log_component is not None:
+            if isinstance(log_component,dict):
+                #Add code to check if the passed dictionary is valid                                                       #TODO
+                # the log level should be of the form debug,info or level_debug,level_info
```

Comment:
did you mean:
```suggestion
                # the log level should be of the form debug|info or level_debug|level_info
```
also, level_ components should be decomposed by parse_log_component, right?

Reply:
I meant that the code will support both info as well as level_info. So to make it more clear, will this help? 
`The log level should be of the form  info or level_debug|info or level_info where 'level_' prefix indicates the specified log class along with all the log classes above it`

> also, level_ components should be decomposed by parse_log_component, right?

I do not think so. This parsing function is only for the case when the user passes in `log_component` as a dictionary. So the different log components will be different keys. And about the validity of the log components, this was the first point in my email and I agree with @DvdMgr's workaround.

Reply:
> I meant that the code will support both info as well as level_info. So to make it more clear, will this help?
> `The log level should be of the form info or level_debug|info or level_info where 'level_' prefix indicates the specified log class along with all the log classes above it`
I would avoid repeating what is already part of the ns-3 documentation, and simply refer to it.
What I would like to see in sem's documentation is what sem supports. It's still not clear to me from this docstring.